iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -212,12 +212,13 @@ class ConversationListViewController: InputHandlerViewContoller, ConversationLis
         // Determine if this is a reply message or regular message. If it's a reply, select the parent
         // message so we can open the thread experience.
         if let parentMessageId = message.parentMessageId {
-            await messagesCell.scrollToMessage(with:  parentMessageId, animateSelection: true)
-            self.selectedMessageView = messagesCell.getBottomFrontmostCell()?.content
```

Comment:
I got rid of having to manually set the selected message view. Instead we just assume that the front most message is the one that's selected. See the transitionable code below.